packages/components/src/hooks/use-copy-to-clipboard.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export const useCopyToClipboard = () => {
+    const [is_copied, setIsCopied] = React.useState(false);
+
+    const copyToClipboard = async text => {
```

Comment:
@Bahar-FS We can return a boolean from this function to determine if copy was successful or not, With the current implementation after first time copying something `is_copied` will always be true and there is no way to differentiate between what has been copied 🤔

Reply:
Can use the implementation here as reference 🙇🏻 
https://usehooks-ts.com/react-hook/use-copy-to-clipboard

Reply:
@farzin-fs Actually I use that as a reference but changed it in a way I liked :)
Let me check again